04_Ansible_Operator_Overview/README.md

```diff
@@ -61,7 +61,7 @@ Example specifying a playbook:
 - version: v1alpha1
   group: foo.example.com
   kind: Foo
-  role: /opt/ansible/playbook.yml
+  role: /opt/ansible/playbook.yaml
```

Comment:
```suggestion
  playbook: /opt/ansible/playbook.yaml
```

Reply:
Another change:

`The Watches file is written in YAML format containing an array of objects. The object has`

Reply:
Also:

`The Watches file is written in YAML format containing an array of objects. The object has`